### PR TITLE
#139 #127 Online wifi Form Backend and faculty profile update working.

### PR DESCRIPTION
**Pull request by RishabhJain2018**

### CODE REVIEW

**Comment by RishabhJain2018** [comment]
#137 #127 Online wifi form backend added and faculty profile update working.

File: HTTP_200/templates/wifi/wifiform.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	<p> 
+	<label> Laptop Mac Address : </label>
+	<input type="text" name="laptop_mac_address" placeholder="Enter Laptop Mac Address">
+	</p>
+	<p>
+	<input type="submit" value="Submit">
+	</p>
+</form>
+</body>
+</html>
```
Comment (DESHRAJ) [inline HTTP_200/templates/wifi/wifiform.html]: @RishabhJain2018 add a newline here.

File: wifi/views.py [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+        if profile:
+            messages.error(request, "Already Registered")
+            return HttpResponseRedirect(reverse("relevent-notice-list"))
+        else:
+            WifiDetail.objects.create(user=user, laptop_mac_address=laptop_mac_address)
+            messages.success(request, "Successfully Registered for Wi-Fi")
+            return HttpResponseRedirect(reverse("relevent-notice-list"))
+
+
+import xlsxwriter
```
Comment (DESHRAJ) [inline wifi/views.py]: Make sure all imports are at the top of the file.

File: wifi/views.py [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+    user=User.objects.get(username=request.user.username)
+    details_student = StudentDetail.objects.get(user=user)
+    details_faculty = FacultyDetail.objects.get(user=user)
+    wifi_detail = WifiDetail.objects.get(user=user)
+    length = wifi_detail.count()
+
+    data = ()
+    for i in xrange(len(length)):
+        pass
```
Comment (DESHRAJ) [inline wifi/views.py]: @RishabhJain2018 why you are iterating and simply doing `pass` ?????

File: wifi/views.py [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+#                 obj.pk,
+#                 obj.laptop_mac_address,
+#         ]
+#         for col_num in xrange(len(row)):
+#             ws.write(row_num, col_num, row[col_num], font_style)
+
+#     wb.save(response)
+#     return response
+
+# export_xls.short_description = u"Export XLS"
```
Comment (DESHRAJ) [inline wifi/views.py]: Please remove comments. :angry:

File: HTTP_200/templates/wifi/wifiform.html (hunk 1 above)
Comment (RishabhJain2018) [inline HTTP_200/templates/wifi/wifiform.html]: @DESHRAJ Sure sir.

File: wifi/views.py (hunk 2 above)
Comment (RishabhJain2018) [inline wifi/views.py]: @DESHRAJ okk sir.

File: wifi/views.py (hunk 3 above)
Comment (RishabhJain2018) [inline wifi/views.py]: @DESHRAJ That I was writing code but I needed to push for @sidjain2901 to work . so I had done that. Well, I will surely remove that sir.

File: wifi/views.py (hunk 4 above)
Comment (RishabhJain2018) [inline wifi/views.py]: @DESHRAJ sorry sir..

**Pull request merged**